corehq/messaging/smsbackends/airtel_tcl/models.py

```diff
@@ -104,9 +104,10 @@ def get_phone_number(phone_number):
         making the request.
         """
         phone_number = strip_plus(phone_number)
-        if phone_number.startswith('91') and len(phone_number) > 2:
+        if re.match(r'^(91)[6-9]{1}\d{9}$', phone_number):
```

Comment:
I think {1} is redundant here, `^(91)[6-9]\d{9}` should do the same?

Reply:
Updated.